Code/Source/svFSI/all_fun.cpp

```diff
@@ -699,8 +723,13 @@ double integ(const ComMod& com_mod, const CmMod& cm_mod, const faceType& lFa, co
   int insd = nsd - 1;
   int tnNo = com_mod.tnNo;
 
+  #ifdef debug_integ_V
+  dmsg << "s.nrows(): " << s.nrows();
+  dmsg << "nsd: " << nsd;
+  #endif
+
   if (s.nrows() != nsd) {
-    throw std::runtime_error("Incompatible vector size in integ");
+    throw std::runtime_error("Incompatible vector size in integV 1");
```

Comment:
Similarly, I do not think that these runtime errors (integV 1 vs. IntegV 2) would be clear for a user. Can we slightly modify the error messages?

Reply:
Adding more descriptive error message. Let me know if this works or if a different message would be better.